opm/output/eclipse/WriteInit.cpp

```diff
@@ -347,6 +422,29 @@ namespace {
         }
     }
 
+    void writeIntegerCellPropertiesLGRCell(const ::Opm::EclipseState&        es,
+                                                   std::vector<int>&         global_fathers,
+                            ::Opm::EclIO::OutputStream::Init&                initFile)
+    {
+        // The INIT file should always contain PVT, saturation function,
+        // equilibration, and fluid-in-place region vectors.
+        // Unconditionally 'get()' these arrays--on a 'const'
+        // FieldPropsManager object--to invoke the automatic creation of the
+        // arrays, and ensure that the keywords exist in the properties
+        // container.
+
+        const auto& fp = es.globalFieldProps();
+        fp.get_int("PVTNUM");
+        fp.get_int("SATNUM");
+        fp.get_int("EQLNUM");
+        fp.get_int("FIPNUM");
+
```

Comment:
You don't need to do this.  The main grid handling already ensures that these properties exist.